ansible_collections/arista/avd/plugins/README.md

```diff
@@ -176,6 +176,32 @@ To use this filter:
 !!! note
     This isn't using the same range syntax as EOS for modular or break-out ports. For example, on EOS `et1/1-2/4` gives you `et1/1, et1/2, et1/3, et1/4, et2/1, et2/2, et2/3, et2/4` on a fixed switch, but a different result on a modular switch depending on the module types. In AVD, the same range would be `et1-2/1-4`.
 
+#### Password filters
+
+The `arista.avd.encrypt` and `arista.avd.decrypt` filters are used to encrypt or decrypt supported passwords.
+It takes as input a `type`
+
+Both take as parameter a value and key
```

Comment:
Please change to this: (suggestion does not work because of code snip)

Usage:

```jinja
{{ <var_with_clear_text_password> | bgp_encrypt(passwd_type=<type>, key=<encryption_key>) }}
{{ <var_with_encrypted_password> | bgp_decrypt(passwd_type=<type>, key=<encryption_key>) }}
```